### PR TITLE
fix: remove plantypes to use remoteconfig directly

### PR DESCRIPTION
**Pull request by rebelchris**
Removed the need for function, just use vars inline

### CODE REVIEW

File: src/routes/webhooks/paddle.ts [hunk 1]
```diff
@@ -36,8 +36,8 @@ const updateUserSubscription = async ({
     return false;
   }
   const subscriptionType = data.data?.items.reduce((acc, item) => {
-    if (item.price?.id && planTypes()[item.price.id]) {
-      acc = planTypes()[item.price.id];
+    if (item.price?.id && remoteConfig.vars?.pricingIds?.[item.price.id]) {
+      acc = remoteConfig.vars.pricingIds?.[item.price.id] ?? '';
```
Comment (rebelchris) [inline src/routes/webhooks/paddle.ts:40]: This was just TS complaining

Comment (capJavert) [reply src/routes/webhooks/paddle.ts:40]: its meant to complain yes 😆 so we can fallback by design

File: src/schema/paddle.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
   Query: {
     pricingIds: (): PricingId => {
       return {
-        value: JSON.stringify(planTypes()),
+        value: JSON.stringify(remoteConfig.vars?.pricingIds),
```
Comment (capJavert) [inline src/schema/paddle.ts:35]: since you defined `value` as required do you want to do `|| {}` here so its always string object?

Comment (capJavert) [reply src/schema/paddle.ts:35]: `JSON.stringify(undefined)` is `undefined`

**Review by capJavert — APPROVED** [review]

File: src/routes/webhooks/paddle.ts (hunk 1 above)
Comment (omBratteng) [reply src/routes/webhooks/paddle.ts:40]: Could define `remoteConfig.vars?.pricingIds?` as a constant in this parent function

Comment (omBratteng) [reply src/routes/webhooks/paddle.ts:40]: `const pricingIds = remoteConfig.vars?.pricingIds || []` maybe

File: src/schema/paddle.ts (hunk 2 above)
Comment (rebelchris) [inline src/schema/paddle.ts:35]:
```suggestion
        value: JSON.stringify(remoteConfig.vars?.pricingIds || {}),
```

**Review by omBratteng — APPROVED** [review]

**Pull request merged**